Backend/dev/migrations/dynamic-offer-driver-app/0562-six-rides-coin-event.sql

```diff
@@ -0,0 +1,41 @@
+UPDATE atlas_driver_offer_bpp.coin_config SET active = False WHERE event_function = 'RidesCompleted 5';
+
+INSERT INTO atlas_driver_offer_bpp.coin_config
+    (id, event_function, event_name, merchant_id, merchant_opt_city_id, coins, expiration_at, active, vehicle_category)
+SELECT
+    md5(random()::TEXT || clock_timestamp()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT)::UUID,
+    'RidesCompleted 6',
+    'EndRide',
+    city.merchant_id,
+    city.id,
+    15,
+    12960000,
+    False,
+    'CAR'
+FROM
+    atlas_driver_offer_bpp.merchant_operating_city AS city;
+
+INSERT INTO atlas_driver_offer_bpp.coin_config
+    (id, event_function, event_name, merchant_id, merchant_opt_city_id, coins, expiration_at, active, vehicle_category)
+SELECT
+    md5(random()::TEXT || clock_timestamp()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT)::UUID,
+    'RidesCompleted 6',
+    'EndRide',
+    city.merchant_id,
+    city.id,
+    15,
+    12960000,
+    False,
+    'AUTO_CATEGORY'
+FROM
+    atlas_driver_offer_bpp.merchant_operating_city AS city;
```

Comment:
UPDATE atlas_driver_offer_bpp.coin_config SET active = False WHERE event_function = 'RidesCompleted 5';

INSERT INTO atlas_driver_offer_bpp.coin_config
    (id, event_function, event_name, merchant_id, merchant_opt_city_id, coins, expiration_at, active, vehicle_category)
SELECT
    md5(random()::TEXT || clock_timestamp()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT || random()::TEXT)::UUID,
    'RidesCompleted 6',
    'EndRide',
    merchant_op_city.merchant_id,
    merchant_op_city.id,
    15,
    12960000,
    True,
    'AUTO_CATEGORY'
FROM
    atlas_driver_offer_bpp.merchant_operating_city AS merchant_op_city;
WHERE
    merchant_op_city.city IN ('Bangalore', 'Mysore','Tumakuru');